Automattic/wp-calypso
# Atomic E2E: add new users for AT Site Editor users.

Related to #74999

## Proposed Changes

This PR adds new users to the user grid and secrets file for FSE-enabled AT users with the goal of mirroring the setup we have for Simple sites currently.

This allows the FSE: Smoke Test spec to pass, as currently it is a permanent failure due to the AT users not having a FSE-enabled theme.

## Testing Instructions

Ensure the following build configurations are passing:
  - [x] Gutenberg E2E atomic (desktop)
  - [x] Gutenberg E2E atomic (mobile)

<img width="879" alt="image" src="https://user-images.githubusercontent.com/6549265/228615188-ccd88916-168f-4ff4-9054-3108ff32743c.png">

## Pre-merge Checklist
